public/components/App.js

```diff
@@ -1,71 +1,85 @@
-import React from "react";
-import Header from "./Header";
-import Panel from "./Panel";
-import Footer from "./Footer";
+import React from 'react'
```

Comment:
Is this some automatic formatter that has been used here? What settings are used?